collector/exporter/otelarrowexporter/README.md

```diff
@@ -26,22 +54,26 @@ Example:
 
 ```yaml
 exporters:
-  otlp:
-    endpoint: otelcol2:4317
+  otelarrow/secure:
+    endpoint: external-collector:4317
     tls:
       cert_file: file.cert
       key_file: file.key
-  otlp/2:
-    endpoint: otelcol2:4317
+  otelarrow/insecure:
+    endpoint: internal-collector:4317
     tls:
       insecure: true
 ```
 
-By default, `gzip` compression is enabled. See [compression comparison](../../config/configgrpc/README.md#compression-comparison) for details benchmark information. To disable, configure as follows:
+By default, `zstd` compression is enabled at the gRPC level. See
+[compression
+comparison](../../config/configgrpc/README.md#compression-comparison)
+for details and benchmark information.  To disable gRPC-level
+compression, configure as follows:
```

Comment:
Maybe silly question - what is the effect of disabling the gRPC-level compression and how does this affect arrow compression results? And how is this different from disabling arrow?